src/tests/ftest/checksum/basic_checksum.py

```diff
@@ -65,10 +67,9 @@ def setUp(self):
 
         self.csum = self.params.get("enable_checksum", '/run/container/*')
         self.container = DaosContainer(self.context)
-        input_param = self.container.cont_input_values
-        input_param.enable_chksum = self.csum
-        self.container.create(poh=self.pool.pool.handle,
-                              con_prop=input_param)
+        input_param = DaosInputParams()
+        input_param.container_params.enable_chksum = self.csum
+        self.container.create(poh=self.pool.pool.handle, input=input_param)
```

Comment:
(pylint-unexpected-keyword-arg) Unexpected keyword argument 'input' in method call

Reply:
Fixed